statsmodels/tsa/tests/test_arima.py

```diff
@@ -759,191 +835,197 @@ def test_arma_predict_mle_dates():
     start, end = 2, 202
     fv = res1.predict(start, end, typ='levels')
-    assert_almost_equal(fv, fc[start:end+1], DECIMAL_4)
+    assert_almost_equal(fv, fc[start:end + 1], DECIMAL_4)
     # start < p, end >nobs 1959q3 - 2015q4
     start, end = 2, 227
     fv = res1.predict(start, end, typ='levels')
-    assert_almost_equal(fv, fc[start:end+1], DECIMAL_4)
+    assert_almost_equal(fv, fc[start:end + 1], DECIMAL_4)
     # start 0, end >0 1960q1 - 1971q4
     start, end = 4, 51
     fv = res1.predict(start, end, typ='levels')
-    assert_almost_equal(fv, fc[start:end+1], DECIMAL_4)
+    assert_almost_equal(fv, fc[start:end + 1], DECIMAL_4)
     # start 0, end nobs 1960q1 - 2009q3
     start, end = 4, 202
     fv = res1.predict(start, end, typ='levels')
-    assert_almost_equal(fv, fc[start:end+1], DECIMAL_4)
+    assert_almost_equal(fv, fc[start:end + 1], DECIMAL_4)
     # start 0, end >nobs 1960q1 - 2015q4
     start, end = 4, 227
     fv = res1.predict(start, end, typ='levels')
-    assert_almost_equal(fv, fc[start:end+1], DECIMAL_4)
+    assert_almost_equal(fv, fc[start:end + 1], DECIMAL_4)
     # start >p, end >0 1965q1 - 1971q4
     start, end = 24, 51
     fv = res1.predict(start, end, typ='levels')
-    assert_almost_equal(fv, fc[start:end+1], DECIMAL_4)
+    assert_almost_equal(fv, fc[start:end + 1], DECIMAL_4)
     # start >p, end nobs 1965q1 - 2009q3
     start, end = 24, 202
     fv = res1.predict(start, end, typ='levels')
-    assert_almost_equal(fv, fc[start:end+1], DECIMAL_4)
+    assert_almost_equal(fv, fc[start:end + 1], DECIMAL_4)
     # start >p, end >nobs 1965q1 - 2015q4
     start, end = 24, 227
     fv = res1.predict(start, end, typ='levels')
-    assert_almost_equal(fv, fc[start:end+1], DECIMAL_4)
+    assert_almost_equal(fv, fc[start:end + 1], DECIMAL_4)
     # start nobs, end nobs 2009q3 - 2009q3
-    #NOTE: raises
-    #start, end = 202, 202
-    #fv = res1.predict(start, end, typ='levels')
-    #assert_almost_equal(fv, [])
+    # NOTE: raises
+    # start, end = 202, 202
+    # fv = res1.predict(start, end, typ='levels')
+    # assert_almost_equal(fv, [])
     # start nobs, end >nobs 2009q3 - 2015q4
     start, end = 202, 227
     fv = res1.predict(start, end, typ='levels')
-    assert_almost_equal(fv, fc[start:end+1], DECIMAL_3)
+    assert_almost_equal(fv, fc[start:end + 1], DECIMAL_3)
     # start >nobs, end >nobs 2009q4 - 2015q4
-    #NOTE: this raises but should not, dynamic forecasts could start
-    #one period out
+    # NOTE: this raises but should not, dynamic forecasts could start
+    # one period out
     start, end = 203, 227
     fv = res1.predict(start, end, typ='levels')
-    assert_almost_equal(fv, fc[start:end+1], DECIMAL_4)
+    assert_almost_equal(fv, fc[start:end + 1], DECIMAL_4)
     # defaults
     start, end = None, None
     fv = res1.predict(start, end, typ='levels')
     assert_almost_equal(fv, fc[1:203], DECIMAL_4)
 
-    #### Dynamic #####
+    # Dynamic
```

Comment:
968-992... not sure what should be done here.  enable, delete, or mark with FIXME

Reply:
They have been enabled.